### PR TITLE
Revert part of 'Improve jobs screen UX'

### PR DESCRIPTION
**Pull request by Visne**
## About the PR
Reverts part of #13657. The new capitalization is inconsistent with the rest of the game, and for that reason looks horrible.

### CODE REVIEW

**Comment by moonheart08** [comment]
Can you just call the fluent captialize function in the jobs menu.
Thanks.

**Comment by Visne** [comment]
You want it capitalized in the job selection UI? In that case it might be better to explicitly lowercase everything in the game, so that in the job selection UI it will still say "Head of Security" instead of "Head of security"

**Comment by EmoGarbage404** [comment]
> You want it capitalized in the job selection UI only? In that case it might be better to explicitly lowercase everything in the game, so that in the job selection UI it will still say "Head of Personnel" instead of "Head of personnel" (unless you consider that to be better)

i think that's what she means. Keep the locale lowercase and then specifically make the capitalized in the ui?

**Comment by moonheart08** [comment]
you'll want a titlecasing function if we don't have one already, fortunately titlecase is easy to define (which is surprising for a language concept)
new fluent function that:
- When it encounters a word not in it's dictionary, capitalizes it
- When it encounters a word in it's dictionary, spits it out unchanged
The dictionary is
- for
- and
- or
- nor
- but
- yet
- so
- by
- in
- of
- for
- on
- to 
(this is just all conjunctions and all the short prepositions)

**Comment by Visne** [comment]
Are you sure you prefer that over keeping the .ftl as they are on master right now, and then lowercasing them in game? That seems a bit more error proof

**Comment by moonheart08** [comment]
> Are you sure you prefer that over keeping the .ftl as they are on master right now, and then lowercasing them in game? That seems a bit more error proof

either works, i just thought TITLECASE() would be more generally useful.

**Comment by Visne** [comment]
I've changed it now so that the jobs in the job picker UI are title case, the department names in the job picker UI lose the "jobs" suffix, and so that the in game job names are lower cased.

**Comment by moonheart08** [comment]
> Okay so now the job names you enter in the ID card computer are uppercase while they will be lowercase in game. I'm beginning to think that maybe your idea is indeed better @moonheart08, but not sure how we would go about localizing a TitleCase() method, any ideas?
> 
> > fortunately titlecase is easy to define (which is surprising for a language concept)
> 
> Not entirely true, there's definitely [some disagreement](https://en.wikipedia.org/wiki/Title_case#Rules) (most importantly, capitalizing the first and last word is a pretty good idea as well)

tl;dr you don't
It's a language function, it can be changed per language.

**Comment by Visne** [comment]
Oh, I only realize now that you mean a function in Fluent, I'm not sure how I missed that the first time I read your comments. Alright, then I will work on that because that seems a lot cleaner in the end.

**Comment by AJCM-git** [comment]
@Visne Status on this?

**Comment by Visne** [comment]
I think I spent a whole bunch of time thinking/asking about the best way to approach this, and then proceeded to forget

So I guess I'll have to figure that out again and then I will fix it Soon™

**Comment by moonheart08** [comment]
@Visne Status?

**Comment by Visne** [comment]
I really want to work on this very soon
